core/src/main/java/nl/nn/adapterframework/jms/JmsListenerBase.java

```diff
@@ -280,14 +282,10 @@ public Message prepareReply(Message rawReply, Map<String,Object> threadContext,
 		return replyMessage;
 	}
 
+	@Deprecated
 	public void setSender(ISender newSender) {
 		sender = newSender;
-		log.debug("["+getName()+"] ** registered sender ["+sender.getName()+"] with properties ["+sender.toString()+"]");
-	}
-
-	@Override
-	public ISender getSender() {
-		return sender;
+		ConfigurationWarnings.add(this, log, "["+getName()+"] has a nested Sender, which is deprecated. Please use attribute replyDestinationName or a Sender nested in Receiver instead", SuppressKeys.DEPRECATION_SUPPRESS_KEY, null);
```

Comment:
Kan dit niet beter als annotatie meegegeven worden?

Reply:
Dat werkte niet. Ik denk dat het alleen maar voor attributen geimplementeerd is.